src/renderer/index.jsx

```diff
@@ -13,19 +13,11 @@ if (process.env.NODE_ENV === 'production') {
     module.hot.accept();
 }
 
-// TODO: enable again, but for the moment seems to conflict with electron-webpack
-// window.eval = global.eval = () => { // eslint-disable-line no-multi-assign, no-eval
-//   import 'bootstrap/dist/css/bootstrap.min.css';
-//     throw new Error(`Sorry, ${remote.app.name} does not support window.eval() for security reasons.`);
-//   };
```

Comment:
we should look into this, well maybe do a full security evaluation that we are not opening any new possibilities for attacks

Reply:
Created a ticket to deal with this properly: https://mattermost.atlassian.net/browse/MM-33828
Some other changes are involved, I might work on this this week.